utils/utf8.cc

```diff
@@ -56,22 +56,30 @@ namespace utils {
 
 namespace utf8 {
 
```

Comment:
Can't comment on commit message so posting here:
s/that **is** spans multiple buffers/that spans multiple buffers

Reply:
fixed